nativelib/src/main/resources/scala-native/delimcc.c

```diff
@@ -15,11 +15,11 @@
 #define ASM_JMPBUF_SIZE 192
 #define JMPBUF_STACK_POINTER_OFFSET (104 / 8)
 #elif defined(__x86_64__) &&                                                   \
-    (defined(__linux__) || defined(__APPLE__)) // x86-64 linux and macOS
+    (defined(__linux__) || defined(__APPLE__) || defined(__FreeBSD__)) // x86-64 linux, macOS and FreeBSD
```

Comment:
I concur with adding `defined(__FreeBSD__)`.  

I checked my October 14 private porting and had made the same change.

Changing the comment is consistent with prior art (good!).
It is author's choice here. I deleted the entire comment rather that adding FreeBSD because the comment adds
no information beyond what the code says pretty explicitly
(due to good coding sytle).